.changeset/gorgeous-kiwis-drive.md

```diff
@@ -0,0 +1,13 @@
+---
+'xstate': major
```

Comment:
```suggestion
'xstate': patch
```

Reply:
Fine by me, but it COULD break compilation of machine configs that currently invoke subscribables that output non-events.

Reply:
Sure, but they probably don't work at runtime either. Quite frankly - every change is potentially a breaking change for somebody. Semver must be applied with some dose of pragmatism ;p